### PR TITLE
#1631 Issue: Replace 'Following' button with a switch

### LINKED ISSUE #1631: Replace 'Following' button with a switch
**Is your feature request related to a problem? Please describe.**
It's very annoying when a button is not clear. Right now the 'Following' button has the action color:
image
... but when I unfollow it has a near-identical appearance:  
image

**Describe the solution you'd like**
Replace the "Following" button: 
image
... with a switch:
image
+ to make the switch make sense change the title of the modal to 'Following' instead of participants.

**Describe alternatives you've considered**
None.

**Additional context**
None.

### PR DESCRIPTION
**Pull request by siva9495**
Resolves #1631 
Modified title of modal Participants to Following.
Checked dark mode is OK.

### CODE REVIEW

**Comment by tijlleenders** [comment]
@siva9495 This is great, thank you!

There is one little detail: Upon initial opening the toggle state does not sync with the database info:
image

After I toggle once, they are in sync. Is that something you could investigate?

**Comment by siva9495** [comment]
@tijlleenders  I have made changes as mentioned

**Review by Tushar-4781 — CHANGES_REQUESTED** [review]
hi @siva9495 i really appreciate your efforts here. Unfortunately, i would like to discard this solution as we are already using antd library. you can use the switch component from the same. https://ant.design/components/switch you can pick the code and modify it as per the description written in the issue

**Comment by siva9495** [comment]
HI @Tushar-4781 
As mentioned above updated the code using the antd library for the toggle switch.

File: package.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
     ]
   },
   "dependencies": {
+    "@ant-design/icons": "^5.2.6",
```
Comment (Tushar-4781) [inline package.json:11]: @siva9495 i don't think its right time to add this dependency. you can use the existing svgs for this. if you open the actions popup my pressing on the circle you will find the two icons. use them instead :) 
image

**Review by Tushar-4781 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (siva9495) [reply package.json:11]: @Tushar-4781 I have a doubt regarding whether the SVG needs to have the same color or be white. Please confirm.

Comment (Tushar-4781) [reply package.json:11]: its better to use the svgs like we are using in other components. i don't understand the sizing issue you mentioned. can you put some ss of code or output. thanks.

Comment (siva9495) [reply package.json]: @Tushar-4781 , sorted it. Removed the library and fixed it using SVGs for the icons. Can you please confirm?

**Review by Tushar-4781 — APPROVED** [review]

**Comment by tijlleenders** [comment]
Thanks @siva9495 glad we got your contribution in!

**Pull request merged**